src/azure/cli/_output.py

```diff
@@ -78,6 +54,20 @@ def format_text(obj):
     except TypeError:
         return ''
 
+def format_table(obj):
+    result = obj.result
+    try:
+        if obj.simple_output_query and not obj.is_query_active:
```

Comment:
I still recommend this, but you can address it (or not) in the next PR since you'll be working in this vicinity anyways. :)

---
In reply to: [76473993](https://github.com/Azure/azure-cli/pull/748#discussion_r76473993) [](ancestors = 76473993)